message_adapter/message_adapter.py

```diff
@@ -112,21 +115,23 @@ def __loadRemoteEvent(self, event):
                 remote_event = json.loads(data['Body'].read().decode('utf-8'))
                 replacement_targets = parsed_json_path.find(event)
                 if not replacement_targets or len(replacement_targets) != 1:
-                    raise Exception('Remote event configuration target {} invalid'.format(target_json_path))
+                    raise Exception(f'Remote event configuration target {target_json_path} invalid')
```

Comment:
Is the `f` before the exception message extraneous?

Reply:
That's the f-string introduced in python 3.6+, it denotes the string is going to interpolate `{variables}`